package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "landuseplanning-api",
-  "version": "1.11.0",
+  "version": "1.12.0",
```

Comment:
Out of cautiousness, you might want to check that changing the models won't break version 1 API compatibility.  Maybe I'm completely off base but it could be worth testing.